### PR TITLE
2.0

### PR DESCRIPTION
**Pull request by SimonPaidla**
- Package does now require neos/fusion-form >= 1.0.x 
- Fixed bug where Package.php class throws an error while installing the package

### CODE REVIEW

**Review by regniets — APPROVED** [review]
OK, seems legitimate.

File: Classes/Controller/BackendModuleController.php [hunk 1]
```diff
@@ -127,19 +156,22 @@ public function showAvailableDimensionsAction($siteNodeName)
      */
     public function showJobPostingsForSiteAction($siteNodeName, $siteDimension)
     {
+        $flashMessages = $this->controllerContext->getFlashMessageContainer()->getMessagesAndFlush();
+
         $selectedRootNode = $this->getSelectedRootNodeByDimension($siteNodeName, $siteDimension);
         $query = new FlowQuery([$selectedRootNode]);
         $jobPostings = $query->find('[instanceof TechDivision.Jobs:Document.JobPosting]')->get();
 
         $this->view->assignMultiple(array(
```
Comment (regniets) [inline Classes/Controller/BackendModuleController.php]: Pls use only one array notation, i.e. "[]" as in Line 162

File: Classes/Controller/BackendModuleController.php [hunk 2]
```diff
@@ -155,13 +187,50 @@ public function sendJobPostingUrisToGoogleApiAction($siteNodeName, $siteDimensio
...
+                $message = $this->translateById('error.googleServiceException');
+                $this->addFlashMessage('', $message . " \r\n " . $error->getMessage() , Message::SEVERITY_ERROR);
+                $this->info('Error: ' . $message . " \r\n - Status code: " . $error->getCode() . ' - Message: ' . $error->getMessage());
+            }
         }
 
-        $this->view->assignMultiple(array(
-            'googleResponse' => $googleResponse,
-        ));
+        if ($this->showGoogleClientConfiguration) {
```
Comment (regniets) [inline Classes/Controller/BackendModuleController.php]: The attribute here should be rather "logGoogleClientConfiguration" than "showGoogleClientConfiguration" as it only logs. Does the client have some __toString Implementation?

File: Configuration/Settings.yaml [hunk 3]
```diff
@@ -29,11 +29,21 @@ Neos:
           createParentDirectories: true
           maximumLogFileSize: 1048576
           logFilesToKeep: 1
+      updateLogger:
+        backend: Neos\Flow\Log\Backend\FileBackend
+        backendOptions:
+          logFileURL: '%FLOW_PATH_DATA%Logs/GoogleUpdate.log'
```
Comment (regniets) [inline Configuration/Settings.yaml]: I'd rather call the logger "jobIndexingLogger" and the logFile "JobIndexingLog", what do you think and put both types inside that log?

File: Resources/Public/JavaScript/Script.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+(function() {
+    $( document ).ready(function() {
```
Comment (regniets) [inline Resources/Public/JavaScript/Script.js]: Do we really need jquery here? Would prefer not to...

File: Classes/Controller/BackendModuleController.php (hunk 1 above)
Comment (SimonPaidla) [reply Classes/Controller/BackendModuleController.php]: *Updated*

File: Classes/Controller/BackendModuleController.php (hunk 2 above)
Comment (SimonPaidla) [reply Classes/Controller/BackendModuleController.php]: *Updated "showGoogleClientConfiguration" to "logGoogleClientConfiguration"*.  
There is no __toString Implementation.

File: Configuration/Settings.yaml (hunk 3 above)
Comment (SimonPaidla) [reply Configuration/Settings.yaml]: *Updated: There is now one logger "jobIndexingLogger" and one logFile "JobIndexingLog"*

File: Resources/Public/JavaScript/Script.js (hunk 4 above)
Comment (SimonPaidla) [reply Resources/Public/JavaScript/Script.js]: *Updated to JavaScript*

File: README.md [hunk 5]
```diff
@@ -37,7 +37,25 @@ Enable the API call on deletion:
...
     GoogleApi:
-      enableApiCallOnJobDeletion: true
+      options:
+        enableApiCallOnJobDeletion: false
+        ...
+```
+
+After configuration there are a few things you need to do, if you haven't done them already:
+[Prerequisites for the Indexing API](https://developers.google.com/search/apis/indexing-api/v3/prereqs#verify-site) 
+
```
Comment (regniets) [inline README.md:62]: We should add some documentation here that the verification must set the service account as owner for the property

**Review by regniets — CHANGES_REQUESTED** [review]
Just realized the missing documentation, we should include this.

**Review by regniets — APPROVED** [review]

**Pull request merged**